main.go

```diff
@@ -0,0 +1,55 @@
+package main
+
+import (
+	"fmt"
+	"log"
+	"net/http"
+
+	"github.com/gorilla/mux"
```

Comment:
Assuming this just pulls in latest master every time, maybe there's a way to use git submodules with https://golang.org/cmd/go/#hdr-Vendor_Directories so we can control the exact version. But I'm still ignorant of how versioning works